pkg/account/storage/v2/sql/api_key_v2/update_api_key_v2.sql

```diff
@@ -0,0 +1,11 @@
+UPDATE
+    api_key
+SET
+    name = %s,
+    role = %d,
+    disabled = %t,
+    created_at = %d,
+    updated_at = %d
+WHERE
+    id = %s AND
+    environment_namespace = %s
```

Comment:
I changed the placeholder part to variable definition.
Strings are represented by %s, integers by %d, and so on.